views/addTranslation.jade

```diff
@@ -16,7 +16,7 @@ block content
         var availableLanguages = !{JSON.stringify(availableLanguages).replace(/<\//g, "<\\/")}
     script(src='/dist/javascripts/songForm.js')
     script.
-        var translationObject = {title: '', author: '', year: '', lang: 'english', copyright: 'CC0', lyric: []}
+        var translationObject = {title: '', author: '', year: '', lang: 'english', copyright: 'private', lyric: []}
```

Comment:
please make it so that these default song/translation objects are defined in one place. That way any changes that need to be made happen in one place.